.github/workflows/peribolos.yaml

```diff
@@ -15,17 +16,18 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
-      
+
       - name: Generating Peribolos configuration
         run: cd tools && go run peribolosbuilder.go -config=../config > ../peribolos.yaml && cd ..
-        
+
       - name: Printing configuration for safety
         run: cat peribolos.yaml
-        
+
       - name: Preparing credentials
         run: echo ${{ secrets.PERIBOLOS_GH_TOKEN }} >> github-token # a secret with the user token to be used for execution
-        
+
       - name: Run Peribolos
         uses: docker://ghcr.io/dynatrace-innovationlab/peribolos:v0
         with:
+          # TODO add confirm before merge
```

Comment:
does this mean that we could have a pipeline that just tells us what it would change on PRs and then the confirm flag(?) on the merge trigger?

Reply:
Yes, we could - I thought about that too - but we should also limit this workflow to changes within `config` and `tools` - what would be your best suggestion to define this? I still think https://stackoverflow.com/a/63131743/3708208 is somewhat of an ugly solution - but maybe you have a better one?

Reply:
you can define triggers to be limited to certain paths as well. see https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#example-including-paths